# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

geometry_msgs/Pose2D pose
float64 curvature
float64 velocity
